docs/cluster-deployment/stripe.rst

```diff
@@ -0,0 +1,164 @@
+.. _cluster-deployment-stripe:
+
+
+===============================
+Deploy a cluster by credit card
+===============================
+
+In this tutorial, we will provide a step by step guide to deploying a cluster
+from scratch. Our recommended procedure assumes you have completed the
+:ref:`signup process <sign-up>` and have created an organization (either
+through AzureAD setup or through the Amazon Cognito signup process). The
+simplest way to then deploy a cluster quickly is to subscribe to CrateDB Cloud
+using your credit card. Here we will explain how to do so, from the initial
+subscribing to the deployment itself. Our payment processing and billing are
+supported by `Stripe`_.
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+
+
+.. _signup-stripe:
+
+Subscribe to CrateDB Cloud
+==========================
+
+When you have completed the initial signup process, you will see the CrateDB
+Cloud Console organization overview. This normally lists all your subscriptions
+and the clusters that can be deployed in them. In this case, however, you
+probably do not yet have any. You will therefore only see a set of subscription
+options, as well as a popup warning that you need a subscription to deploy
+services on CrateDB Cloud:
+
+.. image:: ../_assets/img/stripe-subscribe.png
+   :alt: CrateDB Cloud Console overview
+
+As you can see, although CrateDB Cloud supports subscribing via the Marketplace
```

Comment:
Perhaps we could simply say "You can either create a new subscription and add your credit card details directly, or use one of the supported cloud partners" or something like that? It feels like we're over-emphasising that there's a "new way" to do it :)

Reply:
You're right, that's a bit OTT